app/views/device_models/_form.haml

```diff
@@ -1,90 +1,62 @@
-= validation_errors @device_model
-
-= form_for(@device_model, html: { multipart: true }) do |f|
+= cdx_form_for(@device_model, html: { multipart: true }) do |f|
   .row
     .col
-      .row
-        .col.pe-4
-          = f.label :institution
-        .col
-          .value= f.object.institution
-
-      .row
-        .col.pe-4
-          = f.label :name
-        .col
-          = f.text_field :name
-
+      = f.form_field :institution, value: f.object.institution
 
+      = f.form_field :name do
+        = f.text_field :name
 
       = f.fields_for :manifest do |manifest_form|
-        .row
-          .col.pe-4
-            = manifest_form.label :definition
-          .col
-            - if !@device_model.new_record? && @device_model.manifest.definition
-              .file-uploaded
-                %label.input.on
-                  = link_to @device_model.manifest.filename, manifest_device_model_path(@device_model)
-                  %a.clear-label{:href => "#"}
-                    = image_tag "ic-cross.png"
-            .upload-new-file
-              = manifest_form.file_field :definition, :class => "inputfile"
-              %label{:for => 'device_model_manifest_attributes_definition'}
-                %span
-                  .icon-upload.icon-blue
-                  Choose a file to upload
-                %a.clear-input{:href => "#"}
-                  = image_tag "ic-cross.png"
-
-      .row
-        .col.pe-4
-          = f.label :setup_instructions
-        .col
-          - if !@device_model.new_record? && @device_model.setup_instructions.present?
+        - # FIXME: This is just to surpress the error message showing up in the
+        - # form (via `form_errors`).
+        - f.field_errors :"manifest.definition"
```

Comment:
What about making this explicit with a nice helper?

```ruby
def FormFieldBuilder.skip_errors(attr_name)
  @object.errors.delete(attr_name.to_sym)
  nil
end
```

Reply:
Yeah, that's a good idea. But such a skip method should probably delete all errors with that prefix path, not just exact matches. I wanted to keep that complexity out of this PR. Merging this is a great improvement and I wouldn't delay it. We can easily refactor it later (that's why I left the `FIXME`).

Reply:
No, we really need to handle nested keys already for use case like this. We should skip all errors starting with the key of the nested form. I'm only explicitly skipping `:"manifest.definition"` for now because that's the only error that shows up in CI. If the manifest object contains other errors (and there are a couple of validations on that model), they would still show up in the parent, but we'd want to suppress them and only show them attached to the nested form.